site/content/docs/main/supported-providers.md

```diff
@@ -29,6 +29,7 @@ Contact: [#Velero Slack](https://kubernetes.slack.com/messages/velero), [GitHub
 | [DigitalOcean](https://www.digitalocean.com/)        | DigitalOcean Object Storage  | DigitalOcean Volumes Block Storage | [StackPointCloud](https://github.com/StackPointCloud/ark-plugin-digitalocean)  |                                 |
 | [Hewlett Packard](https://www.hpe.com/us/en/storage.html)     | 🚫                           | HPE Storage                        | [Hewlett Packard](https://github.com/hpe-storage/velero-plugin)  | [Slack](https://slack.hpedev.io/), [GitHub Issue](https://github.com/hpe-storage/velero-plugin/issues) |
 | [OpenEBS](https://openebs.io/)             | 🚫                           | OpenEBS CStor Volume               | [OpenEBS](https://github.com/openebs/velero-plugin)          | [Slack](https://openebs-community.slack.com/), [GitHub Issue](https://github.com/openebs/velero-plugin/issues) |
+| [OpenStack](https://www.openstack.org/) | Swift | Cinder | [Velero OpenStack Plugin Documentation](https://github.com/Lirt/velero-plugin-for-openstack) | [GitHub Issue](https://github.com/Lirt/velero-plugin-for-openstack/issues) |
```

Comment:
nit: I'm thinking if the naming use `OpenStack` would be better?
```suggestion
| [OpenStack](https://www.openstack.org/) | Swift | Cinder | [OpenStack](https://github.com/Lirt/velero-plugin-for-openstack) | [GitHub Issue](https://github.com/Lirt/velero-plugin-for-openstack/issues) |
```

Reply:
Both are OK for me.

Reply:
Updated in https://github.com/vmware-tanzu/velero/pull/3607/commits/f3f4178f78ad47212e0f1ded4408e2ee04e6cab9